invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -252,6 +252,21 @@ <h2 id="files-heading">{{ _('Files') }}</h2>
                               <p>{{ _("Reason") }}: {{ record.access.embargo.reason }}</p>
                             {% endif %}
 
+                            {%- if not current_user.is_anonymous %}
+                            {# TODO allow a similar workflow for guests #}
+                            <hr style="border-color: darkred;">
```

Comment:
styling: in general we consider the inline styling as a bad practice (fe. some stricter content security policies will throw an error), we use the semantic-ui based theme for this, pure html: https://semantic-ui.com/
and react: https://react.semantic-ui.com/,
it will help with removing the multiple paragraphs if you use semantic ui headers and containers

Reply:
yeah, this was just a _very_ quick & dirty hack to make something work